hystrix-contrib/hystrix-javanica/hystrix-javanica-aspectj/src/main/java/com/netflix/hystrix/contrib/javanica/aop/aspectj/AsyncClosureFactory.java

```diff
@@ -36,15 +37,15 @@ public static AsyncClosureFactory getInstance() {
      * {@inheritDoc}.
      */
     @Override
-    boolean isClosureCommand(Object closureObj) {
+    protected boolean isClosureCommand(Object closureObj) {
```

Comment:
`AsyncClosureFactory` extends `AbstractClosureFactory` which is aspectj specific.